### DISCUSSION TITLE
Only "one weekly" meeting

### DISCUSSION THREAD

**Discussion by javiercasares** [WordPress/hosting-handbook#168] [General]
Maybe is not a good idea, but checking the latest meetings, I want to propose changing the meetings to having one but at odd times.

So, the first one will be the 0900UTC meeting, next week the 1800UTC meeting, the next one again the 0900 and go on.

Not many people participates synced, so perhaps without repeating the content, and leaving the conversation there we have more participation.

Moreover, we can add “special meetings” with some announcement at the P2 when we have something special to talk about outside the meetings.

As I said, I don't know if this is a good idea, but it has some logical for me.

Furthermore, with the extra time, we can focus on the projects or have special meetings in the “old hours” for working on tickets and get things done.

**Comment by Crixu** [comment]
I like the idea of having office hours in those weeks we're not having meetings. At least for the APAC and EMEA slot we saw less more contributions over the past months during the meetings. Having such open hours to work together and answer questions might be a way to get more people involved and prepared for Contributor Days

**Reply by chaion07** [reply]
Seconding to @Crixu and having alternate weekly meetings with an office hour makes more sense. Thanks!
